### DISCUSSION TITLE
Show item charges in inventory viewer

### DISCUSSION THREAD

**Discussion by heretohelp11** [runelite/runelite#15843] [Q&A] [answered]
the plugin Inventory viewer is currently not showing the charges on my dodgy necklaces so guessing it doesn't show charges for other items as well. This needs to be fixed as soon as possible, since I like to keep my inventory closed when thieving, and without seeing the charges I cant pickpocket efficiently. Here is an example
https://i.imgur.com/DCei0x0.png

Thanks again.

**Comment by Enriath** [comment, marked as answer]
`Item Charges` already has an option to show equipped item charges as an infobox, along with notifying when certain items fully deplete (such as Dodgy Necklaces)

**Comment by Enriath** [comment]
Otherwise, this feature would be useful for certain other things (such as potion doses or blood essence charges)

**Reply by heretohelp11** [reply]
yes but hate to look at the infobox when the game is spamming uve successfully pickpocketed etc

Indeed will be useful for many other things so hope they implement
